src/examples/typescript/wallet/main.ts

```diff
@@ -46,25 +46,15 @@ async function init(e: Event) {
   // requests to call a JSON-RPC method
   wallet.on('session_request', async (event: Web3WalletTypes.SessionRequest) => {
     // Client logic: prompt user for approval of transaction
-    const { chainId, accountId, method, body } = wallet.parseSessionRequest(event)
-    if (
-      !confirm(
-        `Do you want to proceed with this transaction?: ${JSON.stringify({
-          network: chainId.split(':')[1],
-          accountId,
-          method,
-          body,
-        })}`,
-      )
-    )
-      throw getSdkError('USER_REJECTED_METHODS')
+    const { chainId } = wallet.parseSessionRequest(event)
```

Comment:
we need an end user to confirm whether or not they will accept the proposed transaction